modules/hal_nordic/nrf_802154/CMakeLists.txt

```diff
@@ -67,8 +67,6 @@ target_compile_definitions(zephyr-802154-interface
     # Number of buffers in receive queue.
     NRF_802154_RX_BUFFERS=${CONFIG_NRF_802154_RX_BUFFERS}
 
-    # Enable CSMA/CA
-    NRF_802154_CSMA_CA_ENABLED=1
```

Comment:
Are you sure this is sufficient? The symbol might end up as set anyway: https://github.com/zephyrproject-rtos/hal_nordic/blob/master/drivers/nrf_802154/driver/include/nrf_802154_config.h#L292-L294

Reply:
However you might be right that in https://github.com/zephyrproject-rtos/hal_nordic/blob/master/drivers/nrf_802154/driver/include/nrf_802154_config.h#L292-L294 CSMA-CA should be disabled by default

Reply:
OK, I understand now.